_opcodes/op233.html

```diff
@@ -10,72 +10,319 @@
+			<th>BG:EE / IWD:EE</th>
+			<th>BG2:EE</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td>89</td>
+			<td>PROFICIENCYBASTARDSWORD</td>
+			<td>PROFICIENCYBASTARDSWORD</td>
+			<td>PROFICIENCYBASTARDSWORD</td>
+		</tr>
+		<tr>
+			<td>90</td>
+			<td>PROFICIENCYLONGSWORD</td>
+			<td>PROFICIENCYLONGSWORD</td>
+			<td>PROFICIENCYLONGSWORD</td>
+		</tr>
+		<tr>
+			<td>91</td>
+			<td>PROFICIENCYSHORTSWORD</td>
+			<td>PROFICIENCYSHORTSWORD</td>
+			<td>PROFICIENCYSHORTSWORD</td>
+		</tr>
+		<tr>
+			<td>92</td>
+			<td>PROFICIENCYAXE</td>
+			<td>PROFICIENCYAXE</td>
+			<td>PROFICIENCYAXE</td>
+		</tr>
+		<tr>
+			<td>93</td>
+			<td>PROFICIENCYTWOHANDEDSWORD</td>
+			<td>PROFICIENCYTWOHANDEDSWORD</td>
+			<td>PROFICIENCYTWOHANDEDSWORD</td>
+		</tr>
+		<tr>
+			<td>94</td>
+			<td>PROFICIENCYKATANA</td>
+			<td>PROFICIENCYKATANA</td>
+			<td>PROFICIENCYKATANA</td>
+		</tr>
+		<tr>
+			<td>95</td>
+			<td>PROFICIENCYSCIMITARWAKISASHININJATO</td>
+			<td>PROFICIENCYSCIMITARWAKISASHININJATO</td>
+			<td>PROFICIENCYSCIMITARWAKISASHININJATO</td>
+		</tr>
+		<tr>
+			<td>96</td>
+			<td>PROFICIENCYDAGGER</td>
+			<td>PROFICIENCYDAGGER</td>
+			<td>PROFICIENCYDAGGER</td>
+		</tr>
+		<tr>
+			<td>97</td>
+			<td>PROFICIENCYWARHAMMER</td>
+			<td>PROFICIENCYWARHAMMER</td>
+			<td>PROFICIENCYWARHAMMER</td>
+		</tr>
+		<tr>
+			<td>98</td>
+			<td>PROFICIENCYSPEAR</td>
+			<td>PROFICIENCYSPEAR</td>
+			<td>PROFICIENCYSPEAR</td>
+		</tr>
+		<tr>
+			<td>99</td>
+			<td>PROFICIENCYHALBERD</td>
+			<td>PROFICIENCYHALBERD</td>
+			<td>PROFICIENCYHALBERD</td>
+		</tr>
+		<tr>
+			<td>100</td>
+			<td>PROFICIENCYFLAILMORNINGSTAR</td>
+			<td>PROFICIENCYFLAILMORNINGSTAR</td>
+			<td>PROFICIENCYFLAILMORNINGSTAR</td>
+		</tr>
+		<tr>
+			<td>101</td>
+			<td>PROFICIENCYMACE</td>
+			<td>PROFICIENCYMACE</td>
+			<td>PROFICIENCYMACE</td>
+		</tr>
+		<tr>
+			<td>102</td>
+			<td>PROFICIENCYQUARTERSTAFF</td>
+			<td>PROFICIENCYQUARTERSTAFF</td>
+			<td>PROFICIENCYQUARTERSTAFF</td>
+		</tr>
+		<tr>
+			<td>103</td>
+			<td>PROFICIENCYCROSSBOW</td>
+			<td>PROFICIENCYCROSSBOW</td>
+			<td>PROFICIENCYCROSSBOW</td>
+		</tr>
+		<tr>
+			<td>104</td>
+			<td>PROFICIENCYLONGBOW</td>
+			<td>PROFICIENCYLONGBOW</td>
+			<td>PROFICIENCYLONGBOW</td>
+		</tr>
+		<tr>
+			<td>105</td>
+			<td>PROFICIENCYSHORTBOW</td>
+			<td>PROFICIENCYSHORTBOW</td>
+			<td>PROFICIENCYSHORTBOW</td>
+		</tr>
+		<tr>
+			<td>106</td>
+			<td>PROFICIENCYDART</td>
+			<td>PROFICIENCYDART</td>
+			<td>PROFICIENCYDART</td>
+		</tr>
+		<tr>
+			<td>107</td>
+			<td>PROFICIENCYSLING</td>
+			<td>PROFICIENCYSLING</td>
+			<td>PROFICIENCYSLING</td>
+		</tr>
+		<tr>
+			<td>108</td>
+			<td>PROFICIENCYBLACKJACK</td>
+			<td>PROFICIENCYBLACKJACK</td>
+			<td>PROFICIENCYBLACKJACK</td>
+		</tr>
+		<tr>
+			<td>109</td>
+			<td>PROFICIENCYGUN</td>
+			<td>WEAPON_ENCHANTMENT</td>
+			<td>CLERIC_HALLOW</td>
+		</tr>
+		<tr>
+			<td>110</td>
+			<td>PROFICIENCYMARTIALARTS</td>
+			<td>CLERIC_ARMOR_OF_FAITH</td>
```

Comment:
These anomalies deserve an explanation somewhere, since they're not proficiencies.

Reply:
> These anomalies deserve an explanation somewhere, since they're not proficiencies.

Do you think I should add the following?

- Every `STAT` from `89` to `134` is a weapon proficiency, even the weapon styles.
- Every one of those stats can be set as a weapon's proficiency and it will use that stat's value for proficiency bonuses.
- `CLERIC_ARMOR_OF_FAITH` is a weapon proficiency, being used for an alternative purpose.
- The only thing stopping them from showing up during chargen/level-up proficiency selection is their entries (or lack there of) in `WEAPPROF.2da`. A `NAME_REF` column of `4294967296 (-1)` is hardcoded to hide the proficiency during chargen/level-up. Aside from that it just needs a non-zero maximum for the current class/kit.

Reply:
Looks good, yeah. Maybe add a star to all these non-obvious ones.

Reply:
> Looks good, yeah. Maybe add a star to all these non-obvious ones.

Done.